components/ILIAS/Mail/classes/BackgroundTask/class.ilMassMailTaskProcessor.php

```diff
@@ -80,7 +83,7 @@ public function __construct(
      * @param string $contextId - context ID of the Background task
      * @param array $contextParameters - context parameters for the background tasks
      * @param int $mailsPerTask - Defines how many mails will be added before a background task is executed
-     * @throws ilMailException
+     * @throws ilException
```

Comment:
Do we really need to switch from `ilMailException` to `ilException` here (and in other places)? IMO this should be globally discussed (if we all agree on removing component-specific exceptions or even switch to `SPL` exceptions, which I totally support in general) and if approved, put into a separate PR.